src/main/java/ch/njol/skript/SkriptConfig.java

```diff
@@ -131,27 +131,28 @@ public class SkriptConfig {
 	
 	public static final Option<Boolean> usePlayerUUIDsInVariableNames = new Option<>("use player UUIDs in variable names", false); // TODO change to true later (as well as in the default config)
 	public static final Option<Boolean> enablePlayerVariableFix = new Option<>("player variable fix", true);
-	
-	@SuppressWarnings("null")
-	private static final DateFormat shortDateFormat = DateFormat.getDateTimeInstance(DateFormat.SHORT, DateFormat.SHORT);
-	private static final Option<DateFormat> dateFormat = new Option<>("date format", shortDateFormat, s -> {
+
+	// Required for short serialization in Date class.
+	public static final DateFormat shortDateFormat = DateFormat.getDateTimeInstance(DateFormat.SHORT, DateFormat.SHORT);
+	public static final DateFormat fullDateFormat = DateFormat.getDateTimeInstance(DateFormat.FULL, DateFormat.FULL);
+	public static final Option<DateFormat> dateFormat = new Option<>("date format", shortDateFormat, format -> {
```

Comment:
did you mean to use `fullDateFormat` here? Seems like the default is still the short one.

Reply:
@TheLimeGlass